src/nginxconfig/templates/global_sections/security.vue

```diff
@@ -155,6 +155,24 @@ THE SOFTWARE.
                 </div>
             </div>
         </div>
+
+        <div class="field is-horizontal">
+            <div class="field-label">
+                <label class="label">Block common exploits</label>
```

Comment:
This should be an i18n string please

Reply:
What should I do with languages other than English?

Reply:
You can leave them as English in each of the i18n files with a ` // TODO: translate` comment

Reply:
all done :)